crates/html-macro-test/src/tests/ui.rs

```diff
@@ -0,0 +1,7 @@
+#[test]
```

Comment:
Moved the html-macro-ui test to html-macro-test thanks to https://github.com/dtolnay/trybuild